packages/commonwealth/server/migrations/20220915181216-add-signed-data-fields.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.addColumn(
```

Comment:
Is there any reason this shouldn't be its own table, rather than attaching it directly to each object?